src/Microsoft.SqlTools.ServiceLayer/Localization/sr.cs

```diff
@@ -11341,4950 +11341,6457 @@ public static string ServiceProviderNotSet
             }
         }
 
-        public static string ConnectionServiceListDbErrorNotConnected(string uri)
+        public static string autogrowth_dialog_title
         {
-            return Keys.GetString(Keys.ConnectionServiceListDbErrorNotConnected, uri);
+            get
+            {
+                return Keys.GetString(Keys.autogrowth_dialog_title);
+            }
         }
 
-        public static string ConnectionServiceDbErrorDefaultNotConnected(string uri)
+        public static string autogrowth_dialog_filestreamtitle

```

Comment:
Is there a reason for the name to be all in small letters? Instead of using a camel case? Thanks

Reply:
This file is auto generated (all the SR ones are), so I think git gets a bit confused when rendering it. This string still exists lower in the file.

Reply:
I imported all of these strings from SSMS, so that's the convention that gets used there. I kept the casing for the string names to prevent issues when we import SSMS code in the future, so that we wouldn't need to convert every resource string name. Especially since a lot of resource manager calls in other SQL Tools files use a string literal for the resource name rather than an SR property, so there wouldn't be any compiler errors if a name was changed and not updated where it was used.